code4me-vsc-plugin/package-lock.json

```diff
@@ -1,6 +1,6 @@
 {
 	"name": "code4me-plugin",
-	"version": "1.0.5",
+	"version": "1.0.7",
```

Comment:
oei

Reply:
This will still change for when it's uploaded on main and the version number will be adjusted for the pipeline